ledger-service/http-json-perf/BUILD.bazel

```diff
@@ -13,96 +13,109 @@ scalacopts = lf_scalacopts + [
     "-P:wartremover:traverser:org.wartremover.warts.NonUnitStatements",
 ]
 
-da_scala_library(
-    name = "http-json-perf",
-    srcs = glob(["src/main/scala/**/*.scala"]),
-    plugins = [
-        "@maven//:org_typelevel_kind_projector_{}".format(scala_version_suffix),
-    ],
-    resources = glob(["src/main/resources/**/*"]),
-    scala_deps = [
-        "@maven//:com_github_scopt_scopt",
-        "@maven//:com_typesafe_akka_akka_actor",
-        "@maven//:com_typesafe_akka_akka_http_core",
-        "@maven//:com_typesafe_akka_akka_stream",
-        "@maven//:com_typesafe_scala_logging_scala_logging",
-        "@maven//:org_scalactic_scalactic",
-        "@maven//:org_scalatest_scalatest",
-        "@maven//:org_scalaz_scalaz_core",
-    ],
-    scalacopts = scalacopts,
-    tags = ["maven_coordinates=com.daml:http-json-perf:__VERSION__"],
-    visibility = ["//visibility:public"],
-    runtime_deps = [
-        "@maven//:ch_qos_logback_logback_classic",
-    ],
-    deps = [
-        "//language-support/scala/bindings-akka",
-        "//ledger-api/rs-grpc-bridge",
-        "//ledger-service/http-json",
-        "//ledger-service/http-json-testing",
-        "//ledger-service/jwt",
-        "//libs-scala/gatling-utils",
-        "//libs-scala/ports",
-        "//libs-scala/postgresql-testing",
-        "//libs-scala/scala-utils",
-        "@maven//:com_fasterxml_jackson_core_jackson_core",
-        "@maven//:com_fasterxml_jackson_core_jackson_databind",
-        "@maven//:io_gatling_gatling_app",
-        "@maven//:io_gatling_gatling_commons",
-        "@maven//:io_gatling_gatling_core",
-        "@maven//:io_gatling_gatling_http",
-        "@maven//:io_gatling_gatling_http_client",
-        "@maven//:org_slf4j_slf4j_api",
-    ],
-) if scala_major_version == "2.12" else None
+[
```

Comment:
I think our daily benchmarks run this. Probably need an updated target name.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/9219/commits/9eeec4193bf92fcc0c2b5dfb56c4970b6de0d11a